langlib/lang.string/src/main/ballerina/string.bal

```diff
@@ -25,8 +25,13 @@ public type RegExp regexp:RegExp;
 
 # Returns the length of the string.
 #
+# ```ballerina
+# string hello = "Hello, World!";
+# int length = hello.length();
+# ```
+#
 # + str - the string
-# + return - the number of characters (code points) in parameter `str`
+# + return - the number of characters (Unicode code points) in parameter `str`
```

Comment:
I think we even used "Unicode code points" in some places already. :)

Reply:
Yes, true. Then adding these examples also create an inconsistency with the content of the spec. Isn't it? Anyway, since we are planning to send these changes to spec repo, it will be eventually consistent. But, I got your point. Let's check James's thought on this.